### PR TITLE
create empty tabs for connectivity of PA

### PR DESCRIPTION
**Pull request by Trjegul84**
Names of second level tabs in the new connectivity of protected areas tab were reviewed by Cesar.

### CODE REVIEW

File: src/pages/search/drawer/landscape/connectivity/ConnectivityPAActual.jsx [hunk 1]
```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+
+import SearchContext from 'pages/search/SearchContext';
```
Comment (erikasv) [inline src/pages/search/drawer/landscape/connectivity/ConnectivityPAActual.jsx:3]: The context isn't being used

File: src/pages/search/drawer/landscape/connectivity/ConnectivityPAEE.jsx [hunk 2]
```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+
+import SearchContext from 'pages/search/SearchContext';
```
Comment (erikasv) [inline src/pages/search/drawer/landscape/connectivity/ConnectivityPAEE.jsx:3]: context not being used

File: src/pages/search/drawer/landscape/connectivity/ConnectivityPAInTime.jsx [hunk 3]
```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+
+import SearchContext from 'pages/search/SearchContext';
```
Comment (erikasv) [inline src/pages/search/drawer/landscape/connectivity/ConnectivityPAInTime.jsx:3]: context not used

**Review by erikasv — CHANGES_REQUESTED** [review]
In Landscape component yo need to set the default child tab, so the map can be automatically loaded, this is set in the `childMap` state

File: src/pages/search/drawer/landscape/ConnectivityPA.jsx [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+      component: ConnectivityPAInTime,
+    },
+    {
+        label: {
+          id: 'protectedEE',
+          name: 'Por Ecosistemas estratégicos',
+        },
+        component: ConnectivityPAEE,
+      },
+  ];
```
Comment (cazapatamar) [inline src/pages/search/drawer/landscape/ConnectivityPA.jsx:36]: I would use some similar names regarding those we used in human footprint. I suggest: 

- CurrentPAConnectivity
- TimelinePAConnectivity
- CurrentSEPAConnectivity (Not sure if data is based on the last year but i assume it is, due to similarity to first graph in section 1)

**Review by cazapatamar — CHANGES_REQUESTED** [review]

File: src/pages/search/drawer/landscape/ConnectivityPA.jsx (hunk 4 above)
Comment (cazapatamar) [reply src/pages/search/drawer/landscape/ConnectivityPA.jsx:36]: Take into account we always name components, functions, variables, classes, etc in english and the names you proposed sound a little weird. For example, EE references something in spanish.

Comment (Trjegul84) [reply src/pages/search/drawer/landscape/ConnectivityPA.jsx:36]: Names you suggest are for the component name or applies also for id?

Comment (cazapatamar) [reply src/pages/search/drawer/landscape/ConnectivityPA.jsx:36]: I mean component names. For ids in componentsArray you could use a short versión of component name. Check human footprint case.

File: src/pages/search/drawer/landscape/connectivity/ConnectivityPAInTime.jsx (hunk 3 above)
Comment (Trjegul84) [reply src/pages/search/drawer/landscape/connectivity/ConnectivityPAInTime.jsx]: Unused context were removed.

**Comment by Trjegul84** [comment]
> In Landscape component yo need to set the default child tab, so the map can be automatically loaded, this is set in the `childMap` state

I set currentPA as the default child tab for connectivity.

**Review by erikasv — APPROVED** [review]

File: src/pages/search/drawer/Landscape.jsx [hunk 5]
```diff
@@ -4,6 +4,7 @@ import React from 'react';
 import CompensationFactor from 'pages/search/drawer/landscape/CompensationFactor';
 import Forest from 'pages/search/drawer/landscape/Forest';
 import HumanFootprint from 'pages/search/drawer/landscape/HumanFootprint';
+import ConnectivityPA from 'pages/search/drawer/landscape/ConnectivityPA';
```
Comment (cazapatamar) [inline src/pages/search/drawer/Landscape.jsx:7]: I think PAConnectivity is a better name and it is similar to child components

**Review by cazapatamar — CHANGES_REQUESTED** [review]

File: src/pages/search/drawer/landscape/ConnectivityPA.jsx [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+    {
+      label: {
+        id: 'timelinePA',
+        name: 'Histórico',
+      },
+      component: TimelinePAConnectivity,
+    },
+    {
+        label: {
+          id: 'strategicPA',
```
Comment (cazapatamar) [inline src/pages/search/drawer/landscape/ConnectivityPA.jsx:31]: SEPA

**Review by cazapatamar — CHANGES_REQUESTED** [review]

File: src/pages/search/drawer/Landscape.jsx (hunk 5 above)
Comment (Trjegul84) [reply src/pages/search/drawer/Landscape.jsx]: changed!

**Review by cazapatamar — APPROVED** [review]

File: src/pages/search/drawer/landscape/ConnectivityPA.jsx (hunk 4 above)
Comment (Trjegul84) [reply src/pages/search/drawer/landscape/ConnectivityPA.jsx]: Note:
Id names must match with color manager function names.

**Pull request merged**